### PR TITLE
conda=4.4 doesn't like downgrading to conda=4.3

### PR DESCRIPTION
**Pull request by isuruf**
Ignore the errors for now.

cc @jakirkham

### CODE REVIEW

**Comment by jakirkham** [comment]
Sure. Let’s try. :+1:

File: .travis_scripts/create_feedstocks [hunk 1]
```diff
@@ -28,13 +28,16 @@ conda config --set show_channel_urls true
 conda config --set auto_update_conda false
 conda config --set add_pip_as_python_dependency false
 conda config --add channels conda-forge
+
+set +e
```
Comment (mbargull) [inline .travis_scripts/create_feedstocks]: `unset conda` would be better.

**Comment by jakirkham** [comment]
Seems to still fail as it doesn’t install things. Maybe we could switch to the new `conda activate`.

**Comment by mbargull** [comment]
xref: #5241

**Comment by mbargull** [comment]
> Maybe we could switch to the new `conda activate`.

You'll need to update to conda-forge's `conda` to `4.4` first :wink:

**Pull request merged**